### ISSUE TITLE
PEP 3105 Make print a function in docs/mail

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8412]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8412|
|Type|enhancement|
|Created|2016-06-03 21:20:22Z|
|Branch|https://github.com/twisted/twisted/tree/8412-rodrigc-print|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8412 8412
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8412_rodrigc_print 8412-rodrigc-print
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__None None
time__1464988822581662 1464988822581662
changetime__1465943502103170 1465943502103170
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/128

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

all good.

I have fixed the conflict and merged the PR branch.

Waiting for test results. should be green as we don't have tests for the documentation.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 05a3eda7e9eb3ba0475094bcc5b6efcfd5b86d66
```
#!CommitTicketReference repository="" revision="05a3eda7e9eb3ba0475094bcc5b6efcfd5b86d66"
Merge 8412-rodrigc-print: Use print function in docs/mail.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8412
```
